### PR TITLE
cross compilation fix

### PR DESCRIPTION
**Pull request by missinglink**
Fixes https://github.com/pelias/pbf2json/issues/32, caused by a typo! (`GOOS` != `GOOD`)

```
peter: .../pelias/pbf2json (cross_compilation)$ file build/*
build/pbf2json.darwin-x64: Mach-O 64-bit x86_64 executable
build/pbf2json.linux-arm:  ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, not stripped
build/pbf2json.linux-x64:  ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, not stripped
build/pbf2json.win32-x64:  PE32 executable (console) Intel 80386 (stripped to external PDB), for MS Windows
```

I also removed references to `element.Node.Timestamp` and `element.Way.Timestamp` as they were removed in a vendor lib and cause compilation failures when vendor-ing the new version of that lib.

### CODE REVIEW

**Comment by trescube** [comment]
:shipit:

File: compile.sh [hunk 1]
```diff
@@ -4,62 +4,105 @@
 # to cross-compile for different architectures on your host machine.
 # http://dave.cheney.net/2015/08/22/cross-compilation-with-go-1-5
 
+# ensure the compiler exits with status 0
+function assert() {
```
Comment (orangejulius) [inline compile.sh]: We could use `-e` which will cause the bash script to exit with failure if any line fails. see [bash unofficial strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/). Just change the first line of this file to `#!/bin/bash -e` and you can remove all usage of assert. If any individual line should be allowed to fail you can turn it off temporarily.

File: compile.sh [hunk 2]
```diff
@@ -4,62 +4,105 @@
...
+    exit $1;
+  fi
+}
+
+# ensure the files were compiled to the correct architecture
+declare -A matrix
+matrix["build/pbf2json.darwin-x64"]="Mach-O 64-bit x86_64 executable"
+matrix["build/pbf2json.linux-arm"]="ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, not stripped"
+matrix["build/pbf2json.linux-x64"]="ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, not stripped"
+matrix["build/pbf2json.win32-x64"]="PE32 executable (console) Intel 80386 (stripped to external PDB), for MS Windows"
```
Comment (orangejulius) [inline compile.sh]: checking the file type is awesome

**Comment by orangejulius** [comment]
LGTM

**Comment by missinglink** [comment]
merged with admin since LGTM didnt seem to pick up the 2x :+1:

**Comment by missinglink** [comment]
```
$ npm publish
+ pbf2json@4.1.0
```

**Pull request merged**